### ISSUE TITLE
Unable to connect to sshd on GCP

### ISSUE THREAD

**Issue by steeve** [linuxkit/linuxkit#3118]
**Description**

Trying to ssh on a linuxkit on GCP results in a `connection refused`.

**Steps to reproduce the issue:**

Using the following `yml` on GCP:
```
kernel:
  image: linuxkit/kernel:4.14.55
  cmdline: "console=ttyS0"
init:
  - linuxkit/init:3f1e5c32d6ca9e83a7ea7ad9854da2032971a83f
  - linuxkit/runc:v0.5
  - linuxkit/containerd:v0.5
  - linuxkit/ca-certificates:v0.5
onboot:
  - name: sysctl
    image: linuxkit/sysctl:v0.5
  - name: dhcpcd
    image: linuxkit/dhcpcd:v0.5
    command: ["/sbin/dhcpcd", "--nobackground", "-f", "/dhcpcd.conf", "-1"]
services:
  - name: getty
    image: linuxkit/getty:v0.5
    env:
      - INSECURE=true
  - name: acpid
    image: linuxkit/acpid:v0.5
  - name: rngd
    image: linuxkit/rngd:v0.5
  - name: ntpd
    image: linuxkit/openntpd:v0.5
  - name: sshd
    image: linuxkit/sshd:v0.5
    binds:
      - /root/.ssh/authorized_keys:/root/.ssh/authorized_keys:ro
files:
  - path: etc/linuxkit-config
    metadata: yaml
  - path: root/.ssh/authorized_keys
    source: ~/.ssh/id_rsa.pub
    mode: "0600"
trust:
  org:
    - linuxkit
    - library
```

**Describe the results you received:**

Got a connection refused:
```
$ gcloud compute ssh linuxkit
ssh: connect to host xx.xx.xx.xx port 22: Connection refused
ERROR: (gcloud.compute.ssh) [/usr/bin/ssh] exited with return code [255].
```

Even using `nc` from another instance results in the same issue.

The same yml works fine with run on `hyperkit` and `vmnet`.

**Describe the results you expected:**

Successfully connect.

**Comment by steeve** [comment]
Adding the full serial log:
```
SeaBIOS (version 1.8.2-20171012_061934-google)
Total RAM Size = 0x00000000f0000000 = 3840 MiB
CPUs found: 1     Max CPUs supported: 256
found virtio-scsi at 0:3
virtio-scsi vendor='Google' product='PersistentDisk' rev='1' type=0 removable=0
virtio-scsi blksize=512 sectors=2097152 = 1024 MiB
drive 0x000f2290: PCHS=0/0/0 translation=lba LCHS=1024/32/63 s=2097152
Booting from Hard Disk 0...
[    0.000000] Linux version 4.14.55-linuxkit (root@baf0c275cb44) (gcc version 6.4.0 (Alpine 6.4.0)) #1 SMP Sat Jul 14 12:37:09 UTC 2018
[    0.000000] Command line: BOOT_IMAGE=/kernel console=ttyS0 initrd=/initrd.img
[    0.000000] x86/fpu: Supporting XSAVE feature 0x001: 'x87 floating point registers'
[    0.000000] x86/fpu: Supporting XSAVE feature 0x002: 'SSE registers'
[    0.000000] x86/fpu: Supporting XSAVE feature 0x004: 'AVX registers'
[    0.000000] x86/fpu: xstate_offset[2]:  576, xstate_sizes[2]:  256
[    0.000000] x86/fpu: Enabled xstate features 0x7, context size is 832 bytes, using 'standard' format.
[    0.000000] e820: BIOS-provided physical RAM map:
[    0.000000] BIOS-e820: [mem 0x0000000000000000-0x000000000009fbff] usable
[    0.000000] BIOS-e820: [mem 0x000000000009fc00-0x000000000009ffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000000f0000-0x00000000000fffff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000000100000-0x00000000bfff2fff] usable
[    0.000000] BIOS-e820: [mem 0x00000000bfff3000-0x00000000bfffffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000fffbc000-0x00000000ffffffff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000100000000-0x000000012fffffff] usable
[    0.000000] NX (Execute Disable) protection: active
[    0.000000] SMBIOS 2.4 present.
[    0.000000] DMI: Google Google Compute Engine/Google Compute Engine, BIOS Google 01/01/2011
[    0.000000] Hypervisor detected: KVM
[    0.000000] tsc: Using PIT calibration value
[    0.000000] e820: last_pfn = 0x130000 max_arch_pfn = 0x400000000
[    0.000000] x86/PAT: Configuration [0-7]: WB  WC  UC- UC  WB  WP  UC- WT  
[    0.000000] e820: last_pfn = 0xbfff3 max_arch_pfn = 0x400000000
[    0.000000] found SMP MP-table at [mem 0x000f2310-0x000f231f] mapped at [ffffffffff200310]
[    0.000000] Using GB pages for direct mapping
[    0.000000] RAMDISK: [mem 0x7c9fe000-0x7fffffff]
[    0.000000] ACPI: Early table checksum verification disabled
[    0.000000] ACPI: RSDP 0x00000000000F22D0 000014 (v00 Google)
[    0.000000] ACPI: RSDT 0x00000000BFFF3430 000038 (v01 Google GOOGRSDT 00000001 GOOG 00000001)
[    0.000000] ACPI: FACP 0x00000000BFFFCF60 0000F4 (v02 Google GOOGFACP 00000001 GOOG 00000001)
[    0.000000] ACPI: DSDT 0x00000000BFFF3470 0017B2 (v01 Google GOOGDSDT 00000001 GOOG 00000001)
[    0.000000] ACPI: FACS 0x00000000BFFFCF00 000040
[    0.000000] ACPI: FACS 0x00000000BFFFCF00 000040
[    0.000000] ACPI: SSDT 0x00000000BFFF65F0 00690D (v01 Google GOOGSSDT 00000001 GOOG 00000001)
[    0.000000] ACPI: APIC 0x00000000BFFF5D10 00006E (v01 Google GOOGAPIC 00000001 GOOG 00000001)
[    0.000000] ACPI: WAET 0x00000000BFFF5CE0 000028 (v01 Google GOOGWAET 00000001 GOOG 00000001)
[    0.000000] ACPI: SRAT 0x00000000BFFF4C30 0000B8 (v01 Google GOOGSRAT 00000001 GOOG 00000001)
[    0.000000] kvm-clock: Using msrs 4b564d01 and 4b564d00
[    0.000000] kvm-clock: cpu 0, msr 1:2fffc001, primary cpu clock
[    0.000000] kvm-clock: using sched offset of 1023113363 cycles
[    0.000000] clocksource: kvm-clock: mask: 0xffffffffffffffff max_cycles: 0x1cd42e4dffb, max_idle_ns: 881590591483 ns
[    0.000000] Zone ranges:
[    0.000000]   DMA      [mem 0x0000000000001000-0x0000000000ffffff]
[    0.000000]   DMA32    [mem 0x0000000001000000-0x00000000ffffffff]
[    0.000000]   Normal   [mem 0x0000000100000000-0x000000012fffffff]
[    0.000000]   Device   empty
[    0.000000] Movable zone start for each node
[    0.000000] Early memory node ranges
[    0.000000]   node   0: [mem 0x0000000000001000-0x000000000009efff]
[    0.000000]   node   0: [mem 0x0000000000100000-0x00000000bfff2fff]
[    0.000000]   node   0: [mem 0x0000000100000000-0x000000012fffffff]
[    0.000000] Initmem setup node 0 [mem 0x0000000000001000-0x000000012fffffff]
[    0.000000] ACPI: PM-Timer IO Port: 0xb008
[    0.000000] ACPI: LAPIC_NMI (acpi_id[0xff] dfl dfl lint[0x1])
[    0.000000] IOAPIC[0]: apic_id 0, version 17, address 0xfec00000, GSI 0-23
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 5 global_irq 5 high level)
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 9 global_irq 9 high level)
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 10 global_irq 10 high level)
[    0.000000] ACPI: INT_SRC_OVR (bus 0 bus_irq 11 global_irq 11 high level)
[    0.000000] Using ACPI (MADT) for SMP configuration information
[    0.000000] smpboot: Allowing 1 CPUs, 0 hotplug CPUs
[    0.000000] e820: [mem 0xc0000000-0xfffbbfff] available for PCI devices
[    0.000000] Booting paravirtualized kernel on KVM
[    0.000000] clocksource: refined-jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
[    0.000000] random: get_random_bytes called from start_kernel+0x92/0x4cb with crng_init=0
[    0.000000] setup_percpu: NR_CPUS:128 nr_cpumask_bits:128 nr_cpu_ids:1 nr_node_ids:1
[    0.000000] percpu: Embedded 44 pages/cpu @ffff8f78afc00000 s143256 r8192 d28776 u2097152
[    0.000000] Built 1 zonelists, mobility grouping on.  Total pages: 967548
[    0.000000] Kernel command line: BOOT_IMAGE=/kernel console=ttyS0 initrd=/initrd.img
[    0.000000] PID hash table entries: 4096 (order: 3, 32768 bytes)
[    0.000000] Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes)
[    0.000000] Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes)
[    0.000000] Memory: 3717520K/3931716K available (12300K kernel code, 1465K rwdata, 2996K rodata, 1512K init, 632K bss, 214196K reserved, 0K cma-reserved)
[    0.000000] Kernel/User page tables isolation: enabled
[    0.000000] ftrace: allocating 41047 entries in 161 pages
[    0.010000] Hierarchical RCU implementation.
[    0.010000] 	RCU restricting CPUs from NR_CPUS=128 to nr_cpu_ids=1.
[    0.010000] RCU: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=1
[    0.010000] NR_IRQS: 8448, nr_irqs: 256, preallocated irqs: 16
[    0.010000] Console: colour *CGA 80x25
[    0.010000] console [ttyS0] enabled
[    0.010010] tsc: Detected 2300.000 MHz processor
[    0.010645] Calibrating delay loop (skipped) preset value.. 4600.00 BogoMIPS (lpj=23000000)
[    0.011289] pid_max: default: 32768 minimum: 301
[    0.012064] ACPI: Core revision 20170728
[    0.018014] ACPI: 2 ACPI AML tables successfully acquired and loaded
[    0.018991] Security Framework initialized
[    0.020008] Yama: becoming mindful.
[    0.020553] Mount-cache hash table entries: 8192 (order: 4, 65536 bytes)
[    0.021522] Mountpoint-cache hash table entries: 8192 (order: 4, 65536 bytes)
[    0.022842] CPU: Physical Processor ID: 0
[    0.023507] Last level iTLB entries: 4KB 1024, 2MB 1024, 4MB 1024
[    0.024417] Last level dTLB entries: 4KB 1024, 2MB 1024, 4MB 1024, 1GB 4
[    0.025365] Spectre V2 : Mitigation: Full generic retpoline
[    0.026178] Speculative Store Bypass: Vulnerable
[    0.031895] Freeing SMP alternatives memory: 20K
[    0.033700] smpboot: Max logical packages: 1
[    0.034873] x2apic enabled
[    0.036353] Switched APIC routing to physical x2apic.
[    0.039891] ..TIMER: vector=0x30 apic1=0 pin1=0 apic2=-1 pin2=-1
[    0.040000] smpboot: CPU0: Intel(R) Xeon(R) CPU @ 2.30GHz (family: 0x6, model: 0x3f, stepping: 0x0)
[    0.040000] Performance Events: unsupported p6 CPU model 63 no PMU driver, software events only.
[    0.040000] Hierarchical SRCU implementation.
[    0.040000] NMI watchdog: Perf event create on CPU 0 failed with -2
[    0.040000] NMI watchdog: Perf NMI watchdog permanently disabled
[    0.040000] smp: Bringing up secondary CPUs ...
[    0.040000] smp: Brought up 1 node, 1 CPU
[    0.040000] smpboot: Total of 1 processors activated (4600.00 BogoMIPS)
[    0.040000] devtmpfs: initialized
[    0.040000] x86/mm: Memory block size: 128MB
[    0.040410] evm: security.ima
[    0.040851] evm: security.capability
[    0.041486] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 19112604462750000 ns
[    0.042985] futex hash table entries: 256 (order: 2, 16384 bytes)
[    0.044000] NET: Registered protocol family 16
[    0.044858] cpuidle: using governor ladder
[    0.045425] cpuidle: using governor menu
[    0.046147] ACPI: bus type PCI registered
[    0.047023] PCI: Using configuration type 1 for base access
[    0.049582] HugeTLB registered 1.00 GiB page size, pre-allocated 0 pages
[    0.050007] HugeTLB registered 2.00 MiB page size, pre-allocated 0 pages
[    0.051146] ACPI: Added _OSI(Module Device)
[    0.051727] ACPI: Added _OSI(Processor Device)
[    0.052373] ACPI: Added _OSI(3.0 _SCP Extensions)
[    0.053015] ACPI: Added _OSI(Processor Aggregator Device)
[    0.054182] ACPI: Executed 2 blocks of module-level executable AML code
[    0.077467] ACPI: Interpreter enabled
[    0.078056] ACPI: (supports S0 S5)
[    0.078536] ACPI: Using IOAPIC for interrupt routing
[    0.079304] PCI: Using host bridge windows from ACPI; if necessary, use "pci=nocrs" and report a bug
[    0.080422] ACPI: Enabled 16 GPEs in block 00 to 0F
[    0.106707] ACPI: PCI Root Bridge [PCI0] (domain 0000 [bus 00-ff])
[    0.107682] acpi PNP0A03:00: _OSC: OS supports [ASPM ClockPM Segments MSI]
[    0.108879] acpi PNP0A03:00: _OSC failed (AE_NOT_FOUND); disabling ASPM
[    0.109891] acpi PNP0A03:00: fail to add MMCONFIG information, can't access extended PCI configuration space under this bridge.
[    0.110155] PCI host bridge to bus 0000:00
[    0.110792] pci_bus 0000:00: root bus resource [io  0x0000-0x0cf7 window]
[    0.111893] pci_bus 0000:00: root bus resource [io  0x0d00-0xffff window]
[    0.112929] pci_bus 0000:00: root bus resource [mem 0x000a0000-0x000bffff window]
[    0.114021] pci_bus 0000:00: root bus resource [mem 0xc0000000-0xfebfffff window]
[    0.115021] pci_bus 0000:00: root bus resource [bus 00-ff]
[    0.146243] pci 0000:00:01.3: quirk: [io  0xb000-0xb03f] claimed by PIIX4 ACPI
[    0.188052] ACPI: PCI Interrupt Link [LNKA] (IRQs 5 *10 11)
[    0.190074] ACPI: PCI Interrupt Link [LNKB] (IRQs 5 *10 11)
[    0.191856] ACPI: PCI Interrupt Link [LNKC] (IRQs 5 10 *11)
[    0.193441] ACPI: PCI Interrupt Link [LNKD] (IRQs 5 10 *11)
[    0.195205] ACPI: PCI Interrupt Link [LNKS] (IRQs *9)
[    0.212750] SCSI subsystem initialized
[    0.213484] ACPI: bus type USB registered
[    0.214141] usbcore: registered new interface driver usbfs
[    0.214941] usbcore: registered new interface driver hub
[    0.216911] usbcore: registered new device driver usb
[    0.217616] pps_core: LinuxPPS API ver. 1 registered
[    0.218330] pps_core: Software ver. 5.3.6 - Copyright 2005-2007 Rodolfo Giometti <giometti@linux.it>
[    0.219578] PTP clock support registered
[    0.220056] PCI: Using ACPI for IRQ routing
[    0.221115] NetLabel: Initializing
[    0.221943] NetLabel:  domain hash size = 128
[    0.222549] NetLabel:  protocols = UNLABELED CIPSOv4 CALIPSO
[    0.223500] NetLabel:  unlabeled traffic allowed by default
[    0.224428] clocksource: Switched to clocksource kvm-clock
[    0.239814] VFS: Disk quotas dquot_6.6.0
[    0.240590] VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
[    0.241729] FS-Cache: Loaded
[    0.242205] CacheFiles: Loaded
[    0.242693] pnp: PnP ACPI init
[    0.243714] pnp: PnP ACPI: found 7 devices
[    0.250003] clocksource: acpi_pm: mask: 0xffffff max_cycles: 0xffffff, max_idle_ns: 2085701024 ns
[    0.251450] NET: Registered protocol family 2
[    0.252291] TCP established hash table entries: 32768 (order: 6, 262144 bytes)
[    0.253322] TCP bind hash table entries: 32768 (order: 7, 524288 bytes)
[    0.254370] TCP: Hash tables configured (established 32768 bind 32768)
[    0.256035] UDP hash table entries: 2048 (order: 4, 65536 bytes)
[    0.256929] UDP-Lite hash table entries: 2048 (order: 4, 65536 bytes)
[    0.257901] NET: Registered protocol family 1
[    0.258638] pci 0000:00:00.0: Limiting direct PCI/PCI transfers
[    0.259628] Trying to unpack rootfs image as initramfs...
[    1.214268] Freeing initrd memory: 55304K
[    1.214980] PCI-DMA: Using software bounce buffering for IO (SWIOTLB)
[    1.216039] software IO TLB [mem 0xbbff3000-0xbfff3000] (64MB) mapped at [ffff8f783bff3000-ffff8f783fff2fff]
[    1.217894] RAPL PMU: API unit is 2^-32 Joules, 4 fixed counters, 10737418240 ms ovfl timer
[    1.219370] RAPL PMU: hw unit of domain pp0-core 2^-0 Joules
[    1.220185] RAPL PMU: hw unit of domain package 2^-0 Joules
[    1.221074] RAPL PMU: hw unit of domain dram 2^-0 Joules
[    1.221817] RAPL PMU: hw unit of domain pp1-gpu 2^-0 Joules
[    1.224280] audit: initializing netlink subsys (disabled)
[    1.225542] audit: type=2000 audit(1531754183.817:1): state=initialized audit_enabled=0 res=1
[    1.226758] workingset: timestamp_bits=46 max_order=20 bucket_order=0
[    1.228185] squashfs: version 4.0 (2009/01/31) Phillip Lougher
[    1.229109] FS-Cache: Netfs 'cifs' registered for caching
[    1.229875] fuse init (API version 7.26)
[    1.230584] SGI XFS with ACLs, security attributes, no debug enabled
[    1.231795] 9p: Installing v9fs 9p2000 file system support
[    1.232599] FS-Cache: Netfs '9p' registered for caching
[    1.235221] NET: Registered protocol family 38
[    1.235969] Key type asymmetric registered
[    1.236661] Asymmetric key parser 'x509' registered
[    1.237433] Block layer SCSI generic (bsg) driver version 0.4 loaded (major 247)
[    1.238467] io scheduler noop registered
[    1.239030] io scheduler deadline registered (default)
[    1.239761] io scheduler cfq registered
[    1.240450] io scheduler mq-deadline registered
[    1.241263] io scheduler kyber registered
[    1.242026] shpchp: Standard Hot Plug PCI Controller Driver version: 0.4
[    1.243097] hv_vmbus: registering driver hyperv_fb
[    1.243939] input: Power Button as /devices/LNXSYSTM:00/LNXPWRBN:00/input/input0
[    1.245022] ACPI: Power Button [PWRF]
[    1.245607] input: Sleep Button as /devices/LNXSYSTM:00/LNXSLPBN:00/input/input1
[    1.246638] ACPI: Sleep Button [SLPF]
[    1.249038] ACPI: PCI Interrupt Link [LNKC] enabled at IRQ 11
[    1.249893] virtio-pci 0000:00:03.0: virtio_pci: leaving for legacy driver
[    1.254043] ACPI: PCI Interrupt Link [LNKD] enabled at IRQ 10
[    1.254924] virtio-pci 0000:00:04.0: virtio_pci: leaving for legacy driver
[    1.259015] Serial: 8250/16550 driver, 4 ports, IRQ sharing disabled
[    1.281544] 00:03: ttyS0 at I/O 0x3f8 (irq = 4, base_baud = 115200) is a 16550A
[    1.304374] 00:04: ttyS1 at I/O 0x2f8 (irq = 3, base_baud = 115200) is a 16550A
[    1.327263] 00:05: ttyS2 at I/O 0x3e8 (irq = 6, base_baud = 115200) is a 16550A
[    1.350202] 00:06: ttyS3 at I/O 0x2e8 (irq = 7, base_baud = 115200) is a 16550A
[    1.351931] Non-volatile memory driver v1.3
[    1.352751] Hangcheck: starting hangcheck timer 0.9.1 (tick is 180 seconds, margin is 60 seconds).
[    1.355937] brd: module loaded
[    1.359403] loop: module loaded
[    1.372413] scsi host0: Virtio SCSI HBA
[    1.374054] scsi 0:0:1:0: Direct-Access     Google   PersistentDisk   1    PQ: 0 ANSI: 6
[    1.384119] random: fast init done
[    1.385328] VMware PVSCSI driver - version 1.0.7.0-k
[    1.386138] hv_vmbus: registering driver hv_storvsc
[    1.387274] sd 0:0:1:0: Attached scsi generic sg0 type 0
[    1.388187] sd 0:0:1:0: [sda] 2097152 512-byte logical blocks: (1.07 GB/1.00 GiB)
[    1.389446] sd 0:0:1:0: [sda] 4096-byte physical blocks
[    1.390785] tun: Universal TUN/TAP device driver, 1.6
[    1.392782] sd 0:0:1:0: [sda] Write Protect is off
[    1.401513] sd 0:0:1:0: [sda] Write cache: enabled, read cache: enabled, doesn't support DPO or FUA
[    1.411880] VMware vmxnet3 virtual NIC driver - version 1.4.a.0-k-NAPI
[    1.413200] hv_vmbus: registering driver hv_netvsc
[    1.414202] Fusion MPT base driver 3.04.20
[    1.415049] Copyright (c) 1999-2008 LSI Corporation
[    1.415987] Fusion MPT SPI Host driver 3.04.20
[    1.416901] i8042: PNP: PS/2 Controller [PNP0303:KBD,PNP0f13:MOU] at 0x60,0x64 irq 1,12
[    1.418582] i8042: Warning: Keylock active
[    1.420565] serio: i8042 KBD port at 0x60,0x64 irq 1
[    1.421412]  sda: sda1
[    1.421957] serio: i8042 AUX port at 0x60,0x64 irq 12
[    1.422966] hv_vmbus: registering driver hyperv_keyboard
[    1.423964] mousedev: PS/2 mouse device common for all mice
[    1.425054] sd 0:0:1:0: [sda] Attached SCSI disk
[    1.426047] input: PC Speaker as /devices/platform/pcspkr/input/input3
[    1.427752] rtc_cmos 00:00: RTC can wake from S4
[    1.428807] rtc_cmos 00:00: rtc core: registered rtc_cmos as rtc0
[    1.430225] rtc_cmos 00:00: alarms up to one day, 114 bytes nvram
[    1.431546] i2c /dev entries driver
[    1.432352] device-mapper: ioctl: 4.37.0-ioctl (2017-09-20) initialised: dm-devel@redhat.com
[    1.434793] usbcore: registered new interface driver usbhid
[    1.435880] usbhid: USB HID core driver
[    1.436610] hv_utils: Registering HyperV Utility Driver
[    1.437455] hv_vmbus: registering driver hv_util
[    1.438237] hv_vmbus: registering driver hv_balloon
[    1.439285] oprofile: using NMI timer interrupt.
[    1.440543] GACT probability on
[    1.441343] Mirror/redirect action on
[    1.442165] Simple TC action Loaded
[    1.442752] u32 classifier
[    1.443165]     Performance counters on
[    1.444047]     input device check on
[    1.444708]     Actions configured
[    1.445249] Netfilter messages via NETLINK v0.30.
[    1.446016] nfnl_acct: registering with nfnetlink.
[    1.447028] nf_conntrack version 0.5.0 (16384 buckets, 65536 max)
[    1.448288] ctnetlink v0.93: registering with nfnetlink.
[    1.449414] nf_tables: (c) 2007-2009 Patrick McHardy <kaber@trash.net>
[    1.450689] nf_tables_compat: (c) 2012 Pablo Neira Ayuso <pablo@netfilter.org>
[    1.452219] xt_time: kernel timezone is -0000
[    1.453167] ip_set: protocol 6
[    1.453748] IPVS: Registered protocols (TCP, UDP, SCTP, AH, ESP)
[    1.454774] IPVS: Connection hash table configured (size=4096, memory=64Kbytes)
[    1.456629] IPVS: ipvs loaded.
[    1.457205] IPVS: [rr] scheduler registered.
[    1.457937] IPVS: [wrr] scheduler registered.
[    1.458774] IPVS: [lc] scheduler registered.
[    1.459518] IPVS: [wlc] scheduler registered.
[    1.460415] IPVS: [fo] scheduler registered.
[    1.461216] IPVS: [ovf] scheduler registered.
[    1.461965] IPVS: [lblc] scheduler registered.
[    1.462784] IPVS: [lblcr] scheduler registered.
[    1.463696] IPVS: [dh] scheduler registered.
[    1.464512] IPVS: [sh] scheduler registered.
[    1.465342] IPVS: [sed] scheduler registered.
[    1.466036] IPVS: [nq] scheduler registered.
[    1.466818] IPVS: ftp: loaded support on port[0] = 21
[    1.467911] ipip: IPv4 and MPLS over IPv4 tunneling driver
[    1.469197] gre: GRE over IPv4 demultiplexor driver
[    1.470164] ip_tables: (C) 2000-2006 Netfilter Core Team
[    1.471091] ipt_CLUSTERIP: ClusterIP Version 0.8 loaded successfully
[    1.472241] arp_tables: arp_tables: (C) 2002 David S. Miller
[    1.473196] NET: Registered protocol family 10
[    1.474311] Segment Routing with IPv6
[    1.474967] ip6_tables: (C) 2000-2006 Netfilter Core Team
[    1.476232] NET: Registered protocol family 17
[    1.477191] Bridge firewalling registered
[    1.477943] Ebtables v2.0 registered
[    1.478765] 8021q: 802.1Q VLAN Support v1.8
[    1.479447] 9pnet: Installing 9P2000 support
[    1.480370] Key type dns_resolver registered
[    1.481284] AVX2 version of gcm_enc/dec engaged.
[    1.482091] AES CTR mode by8 optimization enabled
[    1.489147] sched_clock: Marking stable (1480320848, 0)->(1873118373, -392797525)
[    1.490854] registered taskstats version 1
[    1.497341] Key type big_key registered
[    1.501091] Key type trusted registered
[    1.504733] Key type encrypted registered
[    1.505599] ima: No TPM chip found, activating TPM-bypass! (rc=-19)
[    1.507022] ima: Allocated hash algorithm: sha256
[    1.508006] evm: HMAC attrs: 0x1
[    1.508943] rtc_cmos 00:00: setting system clock to 2018-07-16 15:16:24 UTC (1531754184)
[    1.512986] Freeing unused kernel memory: 1512K
[    1.530085] Write protecting the kernel read-only data: 18432k
[    1.531818] Freeing unused kernel memory: 2024K
[    1.535155] Freeing unused kernel memory: 1100K
[    1.630280] input: AT Translated Set 2 keyboard as /devices/platform/i8042/serio0/input/input2
[    2.240090] tsc: Refined TSC clocksource calibration: 2299.830 MHz
[    2.241270] clocksource: tsc: mask: 0xffffffffffffffff max_cycles: 0x212694ad381, max_idle_ns: 440795313445 ns

Welcome to LinuxKit

                        ##         .
                  ## ## ##        ==
               ## ## ## ## ##    ===
           /"""""""""""""""""\___/ ===
          {                       /  ===-
           \______ O           __/
             \    \         __/
              \____\_______/

[    2.463711] IPv6: ADDRCONF(NETDEV_UP): eth0: link is not ready
[    2.480075] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
[    2.483326] random: dhcpcd: uninitialized urandom read (112 bytes read)
eth0: eth0: MTU set to 1460
eth0: waiting for carrier
eth0: carrier acquired
DUID 00:04:b2:8a:00:e6:49:06:62:36:ba:52:ed:44:e6:a1:91:5d
eth0: IAID 0a:80:00:03
eth0: adding address fe80::456e:4702:fb54:91b0
eth0: soliciting an IPv6 router
eth0: soliciting a DHCP lease
eth0: offered 10.128.0.3 from 169.254.169.254
eth0: leased 10.128.0.3 for 86400 seconds
eth0: adding host route to 10.128.0.1
eth0: adding default route via 10.128.0.1
eth0: could not detect a useable init system
exiting due to oneshot
dhcpcd exited
```

**Comment by steeve** [comment]
Confirmed, this is a side effect of #3032, closing.
